client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,197 @@
-		const comp = shallow( <Banner { ...props } href={ '/' } disableHref={ true } /> );
-		expect( comp.find( Card ) ).toHaveLength( 1 );
-		expect( comp.find( Card ).props().href ).toBeNull();
+		const { container } = render( <Banner { ...props } href={ '/' } disableHref={ true } /> );
+		expect( container.firstChild ).toHaveClass( 'card' );
+		expect( container.firstChild ).not.toHaveAttribute( 'href', '/' );
 	} );
 
 	test( 'should render Card with href if href prop is passed but disableHref is true and forceHref is true', () => {
-		const comp = shallow(
+		const { container } = render(
 			<Banner { ...props } href={ '/' } disableHref={ true } forceHref={ true } />
 		);
-		expect( comp.find( Card ) ).toHaveLength( 1 );
-		expect( comp.find( Card ).props().href ).toBe( '/' );
+		expect( container.firstChild ).toHaveClass( 'card' );
+		expect( container.firstChild ).toHaveAttribute( 'href', '/' );
 	} );
 
-	test( 'should render Card with no href and CTA button with href if href prop is passed and callToAction is also passed', () => {
-		const comp = shallow( <Banner { ...props } href={ '/' } callToAction="Go WordPress!" /> );
-		expect( comp.find( Card ) ).toHaveLength( 1 );
-		expect( comp.find( Card ).props().href ).toBeNull();
-		expect( comp.find( Card ).props().onClick ).toBeNull();
+	test( 'should render Card with no href and CTA button with href if href prop is passed and callToAction is also passed', async () => {
+		const handleClick = jest.fn();
+		const { container } = render(
+			<Banner { ...props } href={ '/' } callToAction="Go WordPress!" onClick={ handleClick } />
+		);
+
+		await userEvent.click( container.firstChild );
+		await userEvent.click( screen.getByText( 'Go WordPress!' ) );
+
+		expect( handleClick ).toHaveBeenCalledTimes( 1 );
+
+		expect( container.firstChild ).toHaveClass( 'card' );
+		expect( container.firstChild.href ).toBeUndefined();
 
-		expect( comp.find( Button ) ).toHaveLength( 1 );
-		expect( comp.find( Button ).props().href ).toBe( '/' );
-		expect( comp.find( Button ).props().children ).toBe( 'Go\xA0WordPress!' ); //preventwidows adds \xA0 non-breaking space;
-		expect( comp.find( Button ).props().onClick ).toBe( comp.instance().handleClick );
+		expect( screen.getByText( 'Go WordPress!' ) ).toBeVisible();
+		expect( screen.getByText( 'Go WordPress!' ) ).toHaveAttribute( 'href', '/' );
 	} );
 
-	test( 'should render Card with href and CTA button with no href if href prop is passed and callToAction is also passed and forceHref is true', () => {
-		const comp = shallow(
-			<Banner { ...props } href={ '/' } callToAction="Go WordPress!" forceHref={ true } />
+	test( 'should render Card with href and CTA button with no href if href prop is passed and callToAction is also passed and forceHref is true', async () => {
+		const handleClick = jest.fn();
+		const { container } = render(
+			<Banner
+				{ ...props }
+				href={ '/' }
+				callToAction="Go WordPress!"
+				forceHref={ true }
```

Comment:
Nit: `={ true }` is redundant.

```suggestion
				forceHref
```

Reply:
Would it make sense to do this for `disableHref` as well? For one example: 

https://github.com/Automattic/wp-calypso/blob/a7a5a2a07be4a7ae33742664dfba8da4e82c71ba/client/components/banner/test/index.jsx#L143

Reply:
Totally 👍 It's a React synonym, and as such the shorter one is always preferred.